lib/ansible/modules/windows/win_updates.ps1

```diff
@@ -184,11 +187,28 @@ $update_script_block = {
                 Write-DebugLog -msg "Skipping hidden update $($update_info.title)"
                 continue
             }
-        
-            Write-DebugLog -msg "Adding update $($update_info.id) - $($update_info.title)"
-            $updates_to_install.Add($update) > $null
-        
-            $result.updates[$update_info.id] = $update_info
+
+            $post_cat_matched = $False
```

Comment:
Can be simplified with `$post_cat_matches = ($post_categories.Count -eq 0)`

Reply:
Agreed.